articles/third-party/third-ref-eula.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 3rd party software EULA information | UKCloud Ltd
+description: Provides links to EULA information for 3rd party software
+services: third-party
+author: Sue Highmoor
+reviewer:
+lastreviewed: 
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: 3rd party software EULA information
+toc_fullpath: Reference/third-ref-eula.md
+toc_mdlink: third-ref-eula.md
+---
+
+# 3rd party software EULA information
+
+This article provides links to EULA information for the 3rd party software provided by UKCloud.
+
+Product | EULA
+--------|-----
+Cisco AppDynamics | <https://www.appdynamics.com/download-eula/>
+CommVault Complete Backup & Recovery | <https://documentation.commvault.com/commvault/v11_sp9/article?p=features/misc/eula.htm>
+HyTrust DataControl | <https://www.hytrust.com/resources/hytrust-datacontrol-eula/>
+Moogsoft AIOps | <https://www.moogsoft.com/eula/>
+Portworx Enterprise | <https://portworx.com/end-user-license-agreement/>
+XQ Cyber CyberScore | N/A
+
+## Feedback
+
+If you find an issue with this article, click **Improve this Doc** to suggest a change. If you have an idea for how we could improve any of our services, visit [UKCloud Ideas](https://ideas.ukcloud.com). Alternatively, you can contact us at products@ukcloud.com.
```

Comment:
Could be an idea to make this email address a link with `[products@ukcloud.com](mailto:products@ukcloud.com)`

Reply:
Done